src/lib/annotations/doc/DocAnnotator.js

```diff
@@ -454,6 +454,10 @@ class DocAnnotator extends Annotator {
     bindDOMListeners() {
         super.bindDOMListeners();
 
+        if (!this.permissions.canAnnotate) {
```

Comment:
double check that this doesn't block triggering highlight annotations

Reply:
Removing these. For some reason I had thought that the upstream codebase had `if (!this.canAnnotate)`